ci_framework/roles/operator_build/tasks/zuul_info.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Populate operators from zuul info
+  block:
+    - name: Get the PR owner using github api
+      register: pr_owner
+      ansible.builtin.shell: |
```

Comment:
would it be possible to use native ansible? afaik, ansible.builtin.uri may pass the required headers - then, we can extract the json content we need.

Reply:
yeah, much better, and we don't need to use 'jq' Thanks